ts/routes/deck-options/SimulatorModal.svelte

```diff
@@ -0,0 +1,383 @@
+                    on:click={() => {
+                        $config.newPerDay = simulateFsrsRequest.newLimit;
+                        $config.reviewsPerDay = simulateFsrsRequest.reviewLimit;
+                        $config.maximumReviewInterval = simulateFsrsRequest.maxInterval;
+                        $config.desiredRetention = simulateFsrsRequest.desiredRetention;
+                        $newCardsIgnoreReviewLimit =
+                            simulateFsrsRequest.newCardsIgnoreReviewLimit;
+                        $config.reviewOrder = simulateFsrsRequest.reviewOrder;
+                        onPresetChange();
+                    }}
+                >
+                    <!-- {tr.deckConfigApplyChanges()} -->
+                    {"Save to Preset Options"}
+                </button>
+
+                {#if simulating}
+                    {tr.qtMiscProcessing()}
+                {/if}
+
+                <Graph>
+                    <div class="radio-group">
+                        <InputBox>
+                            <label>
+                                <input
+                                    type="radio"
+                                    value={SimulateSubgraph.count}
+                                    bind:group={simulateSubgraph}
+                                />
+                                {tr.deckConfigFsrsSimulatorRadioCount()}
+                            </label>
+                            <label>
+                                <input
+                                    type="radio"
+                                    value={SimulateSubgraph.time}
+                                    bind:group={simulateSubgraph}
+                                />
+                                {tr.statisticsReviewsTimeCheckbox()}
+                            </label>
+                            <label>
+                                <input
+                                    type="radio"
+                                    value={SimulateSubgraph.memorized}
+                                    bind:group={simulateSubgraph}
+                                />
+                                {tr.deckConfigFsrsSimulatorRadioMemorized()}
+                            </label>
+                        </InputBox>
+                    </div>
+
+                    <svg
+                        bind:this={svg}
+                        viewBox={`0 0 ${bounds.width} ${bounds.height}`}
+                    >
+                        <CumulativeOverlay />
+                        <HoverColumns />
+                        <AxisTicks {bounds} />
+                        <NoDataOverlay {bounds} />
+                    </svg>
+
+                    <TableData {tableData} />
+                </Graph>
+            </div>
+        </div>
+    </div>
+</div>
+
+<style>
+    .modal {
+        background-color: rgba(0, 0, 0, 0.5);
+    }
+
+    :global(.modal-xl) {
+        max-width: 100vw;
+    }
+
```

Comment:
```suggestion

    .modal :global(.tick) :global(.text) {
        font-size: 10px !important;
    }

```

You can override the font-size changes for just the simulator graph with this.
Though as you say, I suspect there's a purpose in it being the way it is.

Reply:
@dae, do you know the purpose of this code?

https://github.com/ankitects/anki/blob/8e13e6bfc159a0d73bbec146c75c16613f9a597e/ts/routes/graphs/Graph.svelte#L39-L58

Reply:
Yep, the text can become unreadably small at the smaller widths/breakpoints without those lines:

<img width="478" alt="Screenshot 2025-02-21 at 5 58 37 pm" src="https://github.com/user-attachments/assets/ef90a501-09ec-4814-8cab-522c0848ea7f" />
<img width="466" alt="Screenshot 2025-02-21 at 5 59 06 pm" src="https://github.com/user-attachments/assets/2c505c60-fdf7-4ee0-ad13-f19918100b7e" />